README.md

```diff
@@ -62,3 +62,17 @@ You should see output similar to the following:
  * Debugger PIN: 226-556-590
 ```
 Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser to view the app.
+
+
+For project :7
+ *For running the git action create todo.yml file with 
+    name: To Do app CI & CD
+    on: [push, pull_request]
+    jobs:
+    And so on.
+
+    Also you have create secrets in github to run docker image
+    KEY:
+    TOKEN: 
+
+    To check the test run clik the action tab and clik on the commits
```

Comment:
Couple of typos in this line 🙂 
```suggestion
    To check the test run click the action tab and click on the commits
```

Also, these are useful instructions for adding the tests to Github Actions, but it would be good to provide more general instructions for running the tests in Docker elsewhere, e.g. something like the below, for your app
```
docker build --tag kamol-test --target test .
docker run --env-file .env kamol-test